firefox-ios/firefox-ios-tests/Tests/XCUITests/CookiePersistenceTests.swift

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class CookiePersistenceTests: BaseTestCase {
```

Comment:
@yoanarios we need to have these tests enabled only in the FullFunctional test plan. You should select each test plan in the top left corner and make sure the suite is disabled in all except for full functional. Let me know if you have doubts or if you want me to push those changes here

Reply:
@isabelrios I've disabled in other test plan, let me know if I disabled in the incorrect way